docs/api/configuration.md

```diff
@@ -0,0 +1,81 @@
+# Configuration Files
+
+## Introduction
+
+Pan3D uses JSON files to save an application state for reuse. The UI and the API include access to import and export functions which read and write these configuration files, respectively. This documentation provides guidelines for reading and writing these files manually.
+
+There are four sections available in the configuration file format: `data_origin`, `data_array`, `data_slices`, and `ui`. The values in these sections will be passed to various functions on the current `DatasetBuilder` instance, as referenced in the following sections. See `DatasetBuilder` documentation for details.
+
+## Example
+
+```
+{
+    "data_origin": "https://ncsa.osn.xsede.org/Pangeo/pangeo-forge/noaa-coastwatch-geopolar-sst-feedstock/noaa-coastwatch-geopolar-sst.zarr",
+    "data_array": {
+        "active": "analysed_sst",
+        "x": "lon",
+        "y": "lat",
+        "t": "time",
+        "t_index": 5
+    },
+    "data_slices": {
+        "lat": [
+            -45,
+            45,
+            100
+        ],
+        "lon": [
+            -90,
+            90,
+            100
+        ]
+    },
+    "ui": {
+        "main_drawer": false,
+        "axis_drawer": false,
+        "expanded_coordinates": []
+    }
+}
+```
+
+For more example configuration files, visit our [Examples on Github](https://github.com/Kitware/pan3d/tree/main/examples).
+
+
+## `data_origin` (Required)
+The value for this key should be a string containing a local path or remote URL, referencing a target dataset readable by `xarray.open_dataset`. This value will be passed to `DatasetBuilder.set_dataset_path`.
+
+## `data_array` (Required)
+The value for this key should be a mapping specifying how to interpret the information in the target dataset. The following table describes keys available in this mapping schema.
+
+| Key | Required? | Type | Value Description |
+|-----|-----------|------|-------------------|
+|`active`|YES     |`str` |This should be a name of an array that exists in the current dataset. This value will be passed to `DatasetBuilder.set_data_array_active_name`. |
+|`x`  |NO (default=None)  |`str`|This should be the name of a coordinate that exists in the active data array. This value will be passed to `DatasetBuilder.set_data_array_axis_names`.|
```

Comment:
It doesn't say that it corresponds to a 1D array listing the world coordinate value along X for describing the grid/mesh.

Reply:
Added in [61ee428](https://github.com/Kitware/pan3d/pull/47/commits/61ee4282f4b4d2f3c30216cdc2fd030236fb230f)